### PR TITLE
Fix local testing

### LINKED ISSUE #52: Fix endpoints

### PR DESCRIPTION
**Pull request by brianaqp**
Close #52

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -27,11 +27,14 @@ app.get("/", (req, res) => {
 // Auth routes
 // This route needs to return the user_id and the categories data.
 app.post("/login", isValidLogin);
-// Data routes
+
+// This is for categories
+app.get("/users/:userId/categories", categoriesController.getCategories);
```
Comment (brianaqp) [inline src/server.js:32]: endpoint for testing

File: src/utils/validateModule.js [hunk 2]
```diff
@@ -15,18 +15,18 @@ function validateModule(obj) {
   }
```
Comment (brianaqp) [inline src/utils/validateModule.js:16]: Validation for receiving all the objects, and for the tittle to have a value

**Review by miwhipps — COMMENTED** [review]
All good!

**Comment by miwhipps** [comment]
Approving!

**Review by miwhipps — APPROVED** [review]
Approved!

**Pull request merged**